src/util/constants.js

```diff
@@ -0,0 +1,6 @@
+export const RANK_COMMANDER = "Commander";
+export const RANK_OPERATIVE = "Operative";
+export const RANK_CORPS = "Corps";
+export const RANK_SPECIAL_FORCES = "Special Forces";
+export const RANK_SUPPORT = "Support";
+export const RANK_HEAVY = "Heavy";
```

Comment:
Might want to refactor these into a single object exported since I assume there will be other constants at some point

Reply:
Something like `RANKS :{ COMMANDER: "Commander" , ...}` ?

Reply:
I guess main downside to that is lack of descriptive imports?

Reply:
:retweet:

By nesting them under a single RANK export, it becomes much more expressive (`RANK.COMMANDER` vs `RANK_COMMANDER`) and also creates a clear association between all of these as being the full list of available ranks. Especially important given that this is a generic constants file (which I think is the right approach. Having too many constant files just makes imports a nightmare in the long run). It explicitly enforces discipline to keep all of the ranks nested together, rather than relying on a convention of "i'll just put them together side by side". You have no choice but to keep it clean.

Bonus: It would also make the this list iterable by using Object.keys/Object.values. 😄